extra-security/keepassxc/autobuild/patches/0001-Fedora-fixes-GNOME-quirks-on-Wayland-sessions.patch

```diff
@@ -0,0 +1,29 @@
+--- keepassxc-2.6.6-orig/src/main.cpp	2021-06-14 18:11:55.987803422 -0400
```

Comment:
fixes => fix. I know they referenced it as such in their [spec file](https://src.fedoraproject.org/rpms/keepassxc/blob/rawhide/f/keepassxc.spec#_11) but let's not.